### PR TITLE
Cover/cmds/exp/cbmem: Add tests.

### PR DESCRIPTION
**Pull request by llogen**
Added more tests for coverage.

Signed-off-by: llogen <christoph.lange@9elements.com>

### CODE REVIEW

File: cmds/exp/cbmem/main_linux.go [hunk 1]
```diff
@@ -149,7 +149,7 @@ func parseCBtable(f *os.File, address int64, sz int) (*CBmem, bool, error) {
 				LB_TAG_PLATFORM_BLOB_VERSION:
 				s, err := bufio.NewReader(io.NewSectionReader(r, j, 65536)).ReadString(0)
 				if err != nil {
-					log.Fatalf("Trying to read string for %s: %v", n, err)
+					return nil, false, fmt.Errorf("Trying to read string for %s: %v", n, err)
```
Comment (ChriMarMe) [inline cmds/exp/cbmem/main_linux.go:152]: Trying -> trying

File: cmds/exp/cbmem/main_linux.go [hunk 2]
```diff
@@ -210,11 +210,11 @@ func parseCBtable(f *os.File, address int64, sz int) (*CBmem, bool, error) {
 				cbmem.MainBoard.Record = rec
 				v, err := bufio.NewReader(io.NewSectionReader(r, j+2, 65536)).ReadString(0)
 				if err != nil {
-					log.Fatalf("Trying to read string for %s: %v", n, err)
+					return nil, false, fmt.Errorf("Trying to read string for %s: %v", n, err)
```
Comment (ChriMarMe) [inline cmds/exp/cbmem/main_linux.go:213]: trying

File: cmds/exp/cbmem/main_linux.go [hunk 3]
```diff
@@ -210,11 +210,11 @@ func parseCBtable(f *os.File, address int64, sz int) (*CBmem, bool, error) {
...
 				}
 				p, err := bufio.NewReader(io.NewSectionReader(r, j+2+int64(len(v)), 65536)).ReadString(0)
 				if err != nil {
-					log.Fatalf("Trying to read string for %s: %v", n, err)
+					return nil, false, fmt.Errorf("Trying to read string for %s: %v", n, err)
```
Comment (ChriMarMe) [inline cmds/exp/cbmem/main_linux.go:217]: trying

File: cmds/exp/cbmem/main_linux.go [hunk 4]
```diff
@@ -369,74 +369,85 @@ func DumpMem(f *os.File, cbmem *CBmem, hexdump bool, w io.Writer) error {
...
-
-func main() {
+func cbMem(w io.Writer) error {
 	var err error
-	flag.Parse()
 	if version {
-		fmt.Println("cbmem in Go, including JSON output")
-		os.Exit(0)
+		fmt.Fprintln(w, "cbmem in Go, including JSON output")
+		return err
```
Comment (ChriMarMe) [inline cmds/exp/cbmem/main_linux.go:376]: Was steht in dem error?

**Review by ChriMarMe — APPROVED** [review]
Only some really really schmol nits

File: cmds/exp/cbmem/main_linux.go (hunk 4 above)
Comment (llogen) [reply cmds/exp/cbmem/main_linux.go:376]: nil

Comment (ChriMarMe) [reply cmds/exp/cbmem/main_linux.go:376]: Aber egal was passiert, da steht immer nil drin :(

Comment (llogen) [reply cmds/exp/cbmem/main_linux.go:376]: jep thats exactly what it should do

Comment (ChriMarMe) [reply cmds/exp/cbmem/main_linux.go:376]: ok

**Review by ChriMarMe — APPROVED** [review]

**Pull request closed without merging**